airflow/www/templates/airflow/trigger.html

```diff
@@ -47,22 +129,90 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
+      </textarea>
+      <div class="panel-body">
 
-    <br />
+        <table class="table table-hover table-bordered">
+          <tbody>
+            {% for form_key, form_details in form_fields.items() %}
+            {% if ("section" not in form_details.schema or not form_details.schema.section) and ("const" not in form_details.schema or not form_details.schema.const) %}
+            {{ form_element(form_key, form_details) }}
+            {% endif %}
+            {% endfor %}
+          </tbody>
+        </table>
 
+        {% for form_section, form_items in form_fields.values() | groupby(attribute="schema.section", default="") %}
+        {% if form_section %}
+        <div class="panel-group" id="accordion_section_{{ form_section | lower() | replace(' ', '_') }}">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title">
+                <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion_section_{{ form_section | lower() | replace(' ', '_') }}" href="#accordion_section_{{ form_section | lower() | replace(' ', '_') }}_href"
+                  data-original-title="" title="" id="{{ form_section | lower() | replace(' ', '_') }}_toggle">{{ form_section }}<span class="caret"></span></a>
+              </h4>
+            </div>
+            <div id="accordion_section_{{ form_section | lower() | replace(' ', '_') }}_href" class="panel-collapse collapse">
+              <div class="panel-body">
+                <table class="table table-hover table-bordered">
+                  <tbody>
+                    {% for form_key, form_details in form_fields.items() %}
+                    {% if form_details.schema.section == form_section and not form_details.schema.const %}
+                    {{ form_element(form_key, form_details) }}
+                    {% endif %}
+                    {% endfor %}
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+        {% endif %}
+        {% endfor %}
+
+        <div class="panel-group" id="accordion_json_conf">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title">
+                <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion_json_conf" href="#accordion_json_conf_href" data-original-title="" title="" id="generated_json_toggle">Generated Configuration JSON<span class="caret"></span></a>
+              </h4>
+            </div>
+            <div id="accordion_json_conf_href" class="panel-collapse collapse">
+              <div class="panel-body">
+                <textarea name="conf" id="json">{{ conf }}</textarea>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
Below the form the generated JSON can be seen - if you want to re-use this for future reference:
![image](https://user-images.githubusercontent.com/95105677/214952362-28966bd8-423a-48ce-8440-c309d3e439b0.png)